scapy/contrib/openflow.py

```diff
@@ -16,11 +16,11 @@
 from scapy.fields import *
 from scapy.layers.l2 import *
 from scapy.layers.inet import *
-from scapy.compat import orb
+from scapy.compat import orb, raw
 
 ### If prereq_autocomplete is True then match prerequisites will be
 ### automatically handled. See OFPMatch class.
-prereq_autocomplete = False
+conf.contribs['OPENFLOW'] = {'prereq_autocomplete': True}
```

Comment:
Why did you change the default behavior?

Reply:
Most people won't look at the source code, and will want those packets to be correctly built.
People that really want to send mis-formed packets will notice this